### DISCUSSION TITLE
the partial download did not work as expect

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [wojtekmaj/react-pdf#1870] [Q&A]
On the server side, I config like this:

```
location /tex/static/proj {
        alias /data/k8s/reddwarf-pro/texhub-server-service/project/;
	add_header Accept-Ranges bytes;
	add_header Access-Control-Expose-Headers Accept-Ranges,Content-Range;
	add_header Content-Encoding identity;
        autoindex off;
    }
```

and this is the response:

<img width="646" alt="image" src="https://github.com/user-attachments/assets/d3b592fc-e5a7-4596-9351-8265d2bbd84c">

but the react pdf still download the whole pdf without partial. Am I missing something?

**Comment by javim89** [comment]
Hi @jiangxiaoqiang !

I'm with the same problem. Could you fixed it?
How do you fix?
